File: others/WorksApplication/Worksap Package/workspace/China_Intern/China_Intern_Cpp/src/entities/region/region_mst_dto.cpp
#include "region_mst_dto.h"
#include <iostream>
using namespace std;

RegionMstDTO::RegionMstDTO(long region_id, string name, long manager_id) {
	this->region_id = region_id;
	this->name = name;
	this->manager_id = manager_id;
}

long &RegionMstDTO::get_region_id() {
	return region_id;
}

string &RegionMstDTO::get_name() {
	return name;
}

long &RegionMstDTO::get_manager_id() {
	return manager_id;
}

ostream &operator <<(ostream &out, RegionMstDTO &obj) {
	out << obj.region_id << " " << obj.name << " " << obj.manager_id;
	return out;
}

istream &operator >>(istream &in, RegionMstDTO &obj) {
	in >> obj.region_id >> obj.name >> obj.manager_id;
	return in;
}

File: others/WorksApplication/Worksap Package/workspace/China_Intern/China_Intern_Java/src/jp/co/worksap/intern/entities/region/RegionMstDTO.java
package jp.co.worksap.intern.entities.region;

import jp.co.worksap.intern.entities.ICsvMasterDTO;

public class RegionMstDTO implements ICsvMasterDTO {
	/**
	 * 
	 */
	private static final long serialVersionUID = -3711231132225687846L;
	private Long regionId;
	private String name;
	private Long managerId;

	/**
	 * Default Constructor 
	 * 
	 * @param regionId
	 * @param name
	 * @param managerId
	 */
	public RegionMstDTO(Long regionId, String name, Long managerId) {
		super();
		this.regionId = regionId;
		this.name = name;
		this.managerId = managerId;
	}

	public Long getRegionId() {
		return regionId;
	}

	public String getName() {
		return name;
	}

	public Long getManagerId() {
		return managerId;
	}

}
